python/perspective/setup.py

```diff
@@ -62,9 +62,11 @@ def get_version(file, name="__version__"):
 
 requires_aiohttp = ["aiohttp"]
 
+requires_jupyter = ["jupyterlab>=3.2,<4"]
```

Comment:
As I mentioned in #1915 - [JupyterLab `3.4.4`](https://github.com/jupyterlab/jupyterlab/releases/tag/v3.4.4) (just released) [doesn't work](https://github.com/finos/perspective/runs/7491578729?check_suite_focus=true) and I haven't fixed it, so these bounds will fail on a fresh install right now.

Reply:
You fixed the incompatibility here https://github.com/finos/perspective/pull/1915/files and this PR is currently passing tests with jupyterlab 3.4.4